pkg/nvml/lib.go

```diff
@@ -144,7 +144,7 @@ func (l *library) close() (rerr error) {
 	return nil
 }
 
-// Default all versioned APIs to v1 (to infer the types)
+// BEGIN_UNVERSIONED_FUNCTIONS
```

Comment:
It is common for machine readable comments not have a leading spaces. What about:

```suggestion
//generatecgoapi: BEGIN_UNVERSIONED_FUNCTIONS
```
(and below)